Dikke Ploaten/Controller/SearchViewController.swift

```diff
@@ -102,7 +85,8 @@ class SearchViewController: UITableViewController, UISearchResultsUpdating {
     override func tableView(_ tableView: UITableView, editActionsForRowAt: IndexPath) -> [UITableViewRowAction]? {
         
         let add = UITableViewRowAction(style: .normal, title: "Add") { action, index in
-            let album = self.albums[index.row]
+            var album = self.albums[index.row]
+            album = Album.init(title: album.title, artist: album.artist, cover: album.cover, genre: nil, releaseYear: nil)
```

Comment:
Waarom doe je dit?

Also, niet `Album.init(` gebruiken om de initializer aan te roepen, maar gewoon `Album(`.

Reply:
Een album heeft van nature een titel, uitvoerder en een image, wanneer ik dit echter toevoeg moet ik ook een userID kunnen meegeven zodat die weet bij wie het album behoort. Als ik dit er niet bijzet wordt er ook geen userID meegegeven en wordt die dus niet getoond.

Reply:
Lijkt me een omweg om de constructor aan te roepen en een nieuw object aan te maken, terwijl je ook gewoon dit kan doen: 

```suggestion
            album.userID = Auth.auth().currentUser?.uid
```

Reply:
Ja idd, niet opgekomen. Is deze change nu al gedaan of moet ik dit zelf changen?

Reply:
Nee is niet uitgevoerd, graag nog aanpassen